docs/components/Layout/Layout.jsx

```diff
@@ -11,7 +11,14 @@ import { theme } from '../../../constants';
 import { MainContainer, AppContainer, MdxContainer, LinkText } from './styles';
 
 const Layout = ({ children }) => {
-  const [currentTheme, setTheme] = useState('dark');
+  const [currentTheme, setTheme] = useState(theme.dark);
+
+  useEffect(() => {
+    const storedTheme = sessionStorage.getItem('theme');
+    if (storedTheme !== currentTheme) {
```

Comment:
What if somehow we don't have a theme? Because `sessionStorage.setItem('theme', type);` call only when user click on item in popover.